workspace_integration_test.go

```diff
@@ -506,6 +506,34 @@ func TestWorkspacesCreate(t *testing.T) {
 		}
 	})
 
+	t.Run("with valid auto-apply-run-trigger option", func(t *testing.T) {
+		skipUnlessBeta(t)
```

Comment:
If this is no longer beta, but only available in TFC we should use the `skipIfEnterprise()` helper

Reply:
omg. yeah, thank you!